pkgs/by-name/td/tdlib/package.nix

```diff
@@ -85,6 +92,10 @@ stdenv.mkDerivation {
     ''
     + lib.optionalString (stdenv.hostPlatform.isDarwin && stdenv.hostPlatform.isAarch64) ''
       sed -i "/vptr/d" test/CMakeLists.txt
+    ''
+    + lib.optionalString tde2eOnly ''
+      substituteInPlace CMakeLists.txt \
+        --replace-fail 'NAMESPACE tde2e::' 'NAMESPACE Td::'
     '';
```

Comment:
Shouldn't this be handled upstream?

Reply:
(The linked PRs have all been merged)

Reply:
Yes, should I `fetchpatch` or wait for the next release?